src/screens/MessageScreen.tsx

```diff
@@ -236,6 +236,18 @@ export function MessageScreen<ParamList extends ParamListBase>({
             </TouchableOpacity>
           </Swipeable>
         ))}
+        {!conversations?.length && !isLoading && (
+          <Swipeable>
```

Comment:
What happens if I try to swipe this?

Reply:
Since no swipeable left or right props are passed, nothing happens.